es6.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  extends: ['eslint-config-mm/es5'],
+  parserOptions: {
+    ecmaVersion: 6
+    // Can't use ES6 imports yet in node: https://github.com/nodejs/help/issues/53#issuecomment-165763446
+    // sourceType: 'module'
+  },
+  env: {
+    node: true,
+    es6: true
+  },
+  rules: {
+    'array-callback-return': 1,
+    'arrow-parens': [1, 'always'],
+    'arrow-body-style': [1, 'always'],
+    'arrow-spacing': 1,
+    'no-const-assign': 1,
+    'no-iterator': 1,
+    'no-var': 1,
```

Comment:
I just have to get used to the idea of using let and const instead. I like this one, because let always limits to the block scope rather than the function scope.